sqlbrite/src/main/java/com/squareup/sqlbrite/QueryToOneOperator.java

```diff
@@ -9,11 +9,11 @@
 
 final class QueryToOneOperator<T> implements Observable.Operator<T, SqlBrite.Query> {
   private final Func1<Cursor, T> mapper;
-  private final boolean emitNull;
```

Comment:
You still need this `boolean` but it should probably be called `emitDefault`. `null` is a valid default value and right now there's no way to emit it.